packages/workspace/src/core/file-utils.ts

```diff
@@ -18,14 +19,22 @@ export interface FileData {
   ext: string;
 }
 
-export interface FileChange extends FileData {
-  getChanges: () => any[];
+export interface Modification {
```

Comment:
I think we should either use the word modifictaion or the word change. I prefer change.